src/utils.js

```diff
@@ -12,19 +12,19 @@ exports = module.exports
  * @private
  */
 exports.randomSeqno = () => {
-  return crypto.randomBytes(20).toString('hex')
+  return crypto.randomBytes(20)
 }
 
 /**
  * Generate a message id, based on the `from` and `seqno`.
  *
  * @param {string} from
- * @param {string} seqno
+ * @param {Buffer} seqno
```

Comment:
Need to change the jsdoc for `randomSeqno` as well, as it's now returning a Buffer rather than string.

Reply:
You got my back :D